flow-client cannot be built using latest npm

### Minimal reproducible example
```
cd flow-client
mvn install
```

### Expected behavior
It builds

### Actual behavior
```
[INFO] --- exec-maven-plugin:1.6.0:exec (npm-install) @ flow-client ---
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @vaadin/flow-deps@0.0.1
npm ERR! Found: webpack@4.39.1
npm ERR! node_modules/webpack
npm ERR!   dev webpack@"4.39.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer webpack@"^4.40.0 || ^5.0.0" from esbuild-loader@2.15.1
npm ERR! node_modules/esbuild-loader
npm ERR!   dev esbuild-loader@"2.15.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Versions:
    - Vaadin / Flow version: master
    - Java version: 16
    - OS version: mac

--- a/flow-client/package.json
+++ b/flow-client/package.json
@@ -42,8 +42,8 @@
     "sinon": "7.5.0",
     "sinon-chai": "3.6.0",
     "typescript": "4.0.3",
-    "webpack": "4.39.1",
-    "webpack-cli": "3.3.6",
+    "webpack": "4.46.0",
+    "webpack-cli": "3.3.12",
     "workbox-core": "5.1.4",
     "workbox-precaching": "5.1.4",
     "xhr-mock": "2.5.0"

